client/sites/settings/v2/style.scss

```
@@ -0,0 +1,18 @@
+@import 'calypso/dashboard/app-dotcom/style.scss';
+
+.is-section-site-settings .dashboard-page-layout {
```

This overrides the styles of a dashboard component. Rather than overriding couldn't we apply these styles to our own class?

Maybe that's what our root route component could be for: it can add a classname that we reference here like:

```suggestion
.is-section-site-settings .dashboard-backport-layout {
```

Then our layout won't break when the dashboard page layout component changes its css.

---

I ended up using our own classname to do the overrides.